notion/block.py

```diff
@@ -121,6 +121,11 @@ def add_new(self, block_type, child_list_key=None, **kwargs):
 
         return block
 
+    def add_new_collection_view_block(self, collection):
+        cvb = self.add_new(CollectionViewBlock)
+        cvb.set("collection_id", collection.id)
+        return cvb
```

Comment:
I wasn't sure whether to create a new method, or update `add_new` with special behaviors for the CVB case.